crates/db_connection_pool/src/mysqlpool.rs

```diff
@@ -100,12 +100,10 @@ impl MySQLConnectionPool {
             }
         }
 
-        let default_ssl_opts = SslOpts::default().with_danger_accept_invalid_certs(true);
         let ssl_opts = match ssl_mode {
             "disable" => None,
```

Comment:
Let's use the format of the MySQL ssl-mode enum: https://dev.mysql.com/doc/refman/8.0/en/connection-options.html#option_general_ssl-mode

i.e.

`DISABLED`
`PREFERRED`

Reply:
It can be confusing cause values for postgres dataconnector are different
I think we should be consisted with this

Reply:
Its actually probably better to be consistent with the source data connector.  Then the user can match directly to  the official documentation rather than guessing what the mapping is.  

Nit - "disabled, preferred, required" instead of disable, require, prefer

Reply:
> It can be confusing cause values for postgres dataconnector are different
postgres dataconnector follows postgres `sslmode` parameter https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-PROTECTION

mysql connector should be consistent with  mysql `ssl-mode`